src/p11_hw_feature_type.ml

```diff
@@ -1,2 +1,3 @@
-type t = Pkcs11.CK_HW_FEATURE_TYPE.t
+type t = Pkcs11_CK_ULONG.t
```

Comment:
I'll fix it. That's probably preferable indeed but not really an issue now because other `P11_*` modules need the JSON serialization capabilities offered by `Pkcs11_CK_ULONG`.